src/fn/csv-to-docs.js

```diff
@@ -29,6 +30,14 @@ const execute = () => {
   const jsonDir = `${environment.pathToProject}/json_docs`;
   fs.mkdir(jsonDir);
 
+  if (!fs.isDirectoryEmpty(jsonDir)) {
+    warn(`There are already docs in ${jsonDir}.
+     New json files will be created along side these existing docs.`);
+    if (!userPrompt.keyInYN('Are you sure you want to continue?')) {
+      throw new Error('User aborted execution.');
+    }
+  }
```

Comment:
This `execute` function is already criminally large.  Can you extract this block into a new little function called something like `warnIfDirectoryHasDocs` and just call the function here?

Reply:
updated.